db/api/src/accounts.rs

```diff
@@ -515,6 +515,34 @@ mod tests {
         assert_eq!(acc, db_acc);
     }
 
+    #[async_std::test]
+    async fn test_should_allow_reannouncement() {
```

Comment:
```suggestion
    async fn test_should_allow_reannouncement() -> Box<dyn std::error::Error> {
```
and then `?` the results would be more straightforward to look at and lead to better readable errors.